c/ldv-linux-3.4-simple/32_1_cilled_true-unreach-call_ok_nondet_linux-3.4-32_1-drivers--input--joystick--spaceorb.ko-ldv_main0_sequence_infinite_withcheck_stateful.cil.out.c

```diff
@@ -1843,7 +1843,10 @@ int init_module(void) ;
 void cleanup_module(void) ;
 extern void *dev_get_drvdata(struct device  const  *dev ) ;
 extern int dev_set_drvdata(struct device *dev , void *data ) ;
-extern struct input_dev *input_allocate_device(void) ;
+struct input_dev *input_allocate_device(void) {
+       return kzalloc(sizeof(struct input_dev), 0x10u | 0x40u | 0x80u);
```

Comment:
In `ldv-linux-3.4-simple` benchmarks `kzalloc` is not defined properly (it calls `kmalloc` and then `__kmalloc` which is undefined)